### DISCUSSION TITLE
Percent sign in bib tex cause trouble in coloration

### DISCUSSION THREAD

**Discussion by thomas-saigre** [James-Yu/LaTeX-Workshop#3915] [Q&A] [answered]
I don't really know if it's a bug (and not even if it is due to this extension), but if I type a `bib` file, with a percent sign that should be escaped (with `\%`), the editor keeps it as a comment :
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/60920471/83ed4c4e-7bdd-4ddf-9462-b6529700651f)
and therefore consider an issue of closed brackets.

This only happens if the language selected is BibTeX 
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/60920471/9c52bd75-9918-4547-a960-790142ddaf88)

if I select `LaTeX` the `\%` is correctly considered, but the coloration of the text no longer corresponds to a BibTeX document : 
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/60920471/197287a9-42c6-445c-bb17-88baa9e5d4db)

**Comment by jlelong** [comment, marked as answer]
See https://github.com/jlelong/vscode-latex-basics/issues/60

**Reply by thomas-saigre** [reply]
Oh, I see this issue has been fixed quite recently ! I think the update has not yet been deployed (or gotten from my laptop) because I see that I still have the old version in the file `Bibtex.tmLanguage.json`, I'll just wait the update !
